server/text_generation_server/utils/weights.py

```diff
@@ -37,6 +91,9 @@ def __init__(
         self.dtype = dtype
         self.process_group = process_group
         self.prefix = prefix
+        self.weights_loader = (
+            _DefaultWeightsLoader() if weights_loader is None else weights_loader
```

Comment:
I think we could make it not optional and force callers to create one.


Then `get_quantizer_loader` could becom `get_loader` and simply return the Regular one instead of `None`. Should save a bit a default logic everywhere,no ?

Reply:
Sounds good!

Reply:
Fixed.